docs/metasploit-framework.wiki/Debugging-Metasploit.md

```diff
@@ -0,0 +1,50 @@
+Pry Debugging
```

Comment:
Can we keep this information at the bottom of the `navigating-and-understanding-metasploits-codebase.md` page like it was previously, I think it was a nice logical grouping of concerns before it was extracted out 👀

Reply:
@adfoster-r7 My concern is that isn't really anything to do with code navigation or understanding the codebase at all. Its related to debugging of the code, which is why I split it into a separate section to try keep things grouped together by category better.

Reply:
Moved this into the same file with https://github.com/rapid7/metasploit-framework/pull/18058/commits/173a93937334b0e85d1c97349848de70d86ed85f